pandas/core/frame.py

```diff
@@ -4628,10 +4698,16 @@ def query(self, expr: str, *, inplace: bool = False, **kwargs) -> DataFrame | No
         if not isinstance(expr, str):
             msg = f"expr must be a string to be evaluated, {type(expr)} given"
             raise ValueError(msg)
-        kwargs["level"] = kwargs.pop("level", 0) + 1
-        kwargs["target"] = None
```

Comment:
Looks like target was left out.

Reply:
Correct, I was just debugging it and realize I missed it.